src/test/ui/enum/suggest-default-attribute.rs

```diff
@@ -0,0 +1,7 @@
+pub enum Test {
```

Comment:
We should also suggest changing the attribute to `derive(Default)` if they meant it by annotating a `struct` or `enum` with `#[default]`, although this does not need to be in this PR. could be a followup.